ui/vite.config.ts

```diff
@@ -10,7 +10,7 @@ export default defineConfig({
 		vue(),
 		viteExternalsPlugin({
 			vue: 'Vue',
-      pinia: 'Pinia'
+			pinia: 'Pinia'
```

Comment:
can we add `'vue-router': 'VueRouter'` here and in the globals object below? Also then in the `external` array add `vue-router`. Then rebuild and the .es / .umd files should be a lot small because it will not contain the vue-router package code with it.